include/SFML/Audio/SoundRecorder.hpp

```diff
@@ -145,6 +145,36 @@ class SFML_AUDIO_API SoundRecorder : AlResource
     const std::string& getDevice() const;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Set the channel count of the audio capture device
+    ///
+    /// This method allows you to set the channel count of the
+    /// recording. Currently only 16-bit mono and 16-bit stereo
+    /// are supported.
```

Comment:
"This method[...]" is just a repeated brief explanation. I think it can be dropped. "Currently only[...]" makes sense though.